js/graviex.js

```diff
@@ -0,0 +1,1073 @@
+                        'strategy/create', // Not Using Currently In Unified Methods.
+                        'strategy/update', // Not Using Currently In Unified Methods.
+                        'strategy/activate', // Not Using Currently In Unified Methods.
+                        'strategy/deactivate', // Not Using Currently In Unified Methods.
+                    ],
+                },
+            },
+            'fees': {
+            },
+            'limits': {
+                'amount': {
+                    'min': 0.001,
+                    'max': undefined,
+                },
+            },
+            'precision': {
+                'amount': 8,
+                'price': 8,
+            },
+            'exceptions': {
+                'exact': {
+                    '2002': InvalidOrder, // {"error":{"code":2002,"message":"Failed to create order. Reason: Volume is too small"}}
+                    '1001': ExchangeError, // {"error":{"code":1001,"message":"market does not have a valid value"}}
+                    '2008': AuthenticationError, // {"error":{"code":2008,"message":"The access key XXXXXXXXXXXXXXXXXXXX does not exist."}}
+                },
+            },
+        });
+    }
+
+    async fetchTime (params = {}) {
+        const response = await this.publicGetTimestamp (params);
+        // 1644821226
+        return response;
+    }
+
+    async fetchMarkets (params = {}) {
+        const response = await this.publicGetMarkets ();
+        // {
+        //     "id": "giobtc",
+        //     "name": "GIO/BTC",
+        // };
+        const markets = response;
+        const result = [];
+        for (let i = 0; i < markets.length; i++) {
+            const market = markets[i];
+            const id = this.safeString (market, 'id');
+            const symbol = this.safeString (market, 'name');
+            const [ baseId, quoteId ] = symbol.split ('/');
+            result.push ({
+                'id': id,
+                'symbol': symbol,
+                'base': this.safeCurrencyCode (baseId),
+                'quote': this.safeCurrencyCode (quoteId),
+                'settle': undefined,
+                'baseId': baseId,
+                'quoteId': quoteId,
+                'settleId': undefined,
+                'type': 'spot',
+                'spot': true,
+                'margin': false,
+                'swap': false,
+                'future': false,
+                'option': false,
+                'active': undefined,
+                'contract': false,
+                'linear': undefined,
+                'inverse': undefined,
+                'contractSize': undefined,
+                'expiry': undefined,
+                'expiryDatetime': undefined,
+                'strike': undefined,
+                'optionType': undefined,
+                'precision': this.precision,
+                'limits': this.extend ({
```

Comment:
we shouldn't call `this.extend` here, it's better to just declare the structure without any function calls

Reply:
OKay. Fixed in the latest commit https://github.com/ccxt/ccxt/pull/11697/commits/24b2627beff01a76b81a32b5728efb6846cd8010